### PR TITLE
Remove deprecated component http

### PR DESCRIPTION
**Pull request by tveskrna**

### CODE REVIEW

**Comment by davsclaus** [comment]
We are not removing deprecated components in Camel 2.x at this time.
They are removed in Camel 3 onwards.

So we cannot accept this PR.

**Comment by oscerd** [comment]
But this PR doesn't remove the component. It passes from http(deprecated) to http4

**Comment by davsclaus** [comment]
Ah sorry the title is misleading.

**Comment by oscerd** [comment]
I'll merge this today

File: components/camel-restlet/pom.xml [hunk 1]
```diff
@@ -161,5 +161,21 @@
     </dependency>
 
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
```
Comment (dmvolod) [inline components/camel-restlet/pom.xml]: It would be nice to use maven-jar-plugin-version from the parent/pom.xml

Comment (tveskrna) [reply components/camel-restlet/pom.xml]: Sorry, I removed it, because it was code which shouldn't be here.

**Comment by oscerd** [comment]
Thanks, merged on camel-2.x and camel-2.23.x

**Pull request closed without merging**